VoodooPS2Keyboard/VoodooPS2Keyboard.cpp

```diff
@@ -1923,7 +1923,12 @@ void ApplePS2Keyboard::setKeyboardEnable(bool enable)
     // (keyboard enable/disable command)
     TPS2Request<2> request;
     request.commands[0].command = kPS2C_WriteDataPort;
-    request.commands[0].inOrOut = enable ? kDP_Enable : kDP_SetDefaultsAndDisable;
+    
+    if(_kbd_fixdisable)
+        request.commands[0].inOrOut = enable ? kDP_Enable : kDP_SetDefaults;
```

Comment:
Looking at Linux (https://github.com/torvalds/linux/blob/master/drivers/input/keyboard/atkbd.c#L913), it should be fine to just leave it as SetDefaults. I think it'd be fine to just replace with `kDP_SetDefaults` or to at least have the fix enabled by default.

Reply:
Did not want to change the default behavior of the driver; hence only those with the problem need to set the boot argn.

Reply:
I think it’s fine to change the default behavior. Basically every device I’ve seen is affected by this bug, and the new behavior is what Linux does so it should be fine. Could merge it in and see if there are any reports of this breaking.

Reply:
Done.

"I think it'd be fine to just replace with kDP_SetDefaults."

Reply:
Oops, I spot code to remove, hang on.